<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlySight Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="DataPlot" name="plotArea" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionImportFolder"/>
    <addaction name="separator"/>
    <addaction name="actionImportGates"/>
    <addaction name="actionImportVideo"/>
    <addaction name="separator"/>
    <addaction name="actionExportTrack"/>
    <addaction name="actionExportPlot"/>
    <addaction name="actionExportKML"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlots">
    <property name="title">
     <string>&amp;Left</string>
    </property>
    <addaction name="actionElevation"/>
    <addaction name="separator"/>
    <addaction name="actionHorizontalSpeed"/>
    <addaction name="actionVerticalSpeed"/>
    <addaction name="actionTotalSpeed"/>
    <addaction name="separator"/>
    <addaction name="actionCourse"/>
    <addaction name="actionCourseRate"/>
    <addaction name="actionCourseAccuracy"/>
    <addaction name="separator"/>
    <addaction name="actionGlideRatio"/>
    <addaction name="actionDiveAngle"/>
    <addaction name="actionCurvature"/>
    <addaction name="separator"/>
    <addaction name="actionHorizontalAccuracy"/>
    <addaction name="actionVerticalAccuracy"/>
    <addaction name="actionSpeedAccuracy"/>
    <addaction name="separator"/>
    <addaction name="actionNumberOfSatellites"/>
    <addaction name="separator"/>
    <addaction name="actionAcceleration"/>
    <addaction name="actionTotalEnergy"/>
    <addaction name="actionEnergyRate"/>
    <addaction name="separator"/>
    <addaction name="actionLift"/>
    <addaction name="actionDrag"/>
   </widget>
   <widget class="QMenu" name="menuDomain">
    <property name="title">
     <string>&amp;Bottom</string>
    </property>
    <addaction name="actionTime"/>
    <addaction name="actionDistance2D"/>
    <addaction name="actionDistance3D"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionUndoZoom"/>
    <addaction name="actionRedoZoom"/>
    <addaction name="actionZoomToExtent"/>
    <addaction name="separator"/>
    <addaction name="actionPan"/>
    <addaction name="actionZoom"/>
    <addaction name="actionMeasure"/>
    <addaction name="separator"/>
    <addaction name="actionZero"/>
    <addaction name="actionGround"/>
    <addaction name="actionSetCourse"/>
    <addaction name="separator"/>
    <addaction name="actionWind"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShowLeftView"/>
    <addaction name="actionShowTopView"/>
    <addaction name="actionShowFrontView"/>
    <addaction name="actionShowMapView"/>
    <addaction name="actionShowOrthoView"/>
    <addaction name="separator"/>
    <addaction name="actionShowWindView"/>
    <addaction name="separator"/>
    <addaction name="actionShowScoringView"/>
    <addaction name="actionShowLiftDragView"/>
    <addaction name="separator"/>
    <addaction name="actionShowPlaybackView"/>
    <addaction name="separator"/>
    <addaction name="actionShowLogbookView"/>
   </widget>
   <widget class="QMenu" name="menuTrack">
    <property name="title">
     <string>Track</string>
    </property>
    <addaction name="actionDeleteTrack"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuPlots"/>
   <addaction name="menuDomain"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuTrack"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import &amp;Tracks...</string>
   </property>
   <property name="toolTip">
    <string>Import Tracks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionElevation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Elevation</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionVerticalSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Vertical Speed</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionHorizontalSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Horizontal Speed</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionMetric">
   <property name="text">
    <string>&amp;Metric</string>
   </property>
  </action>
  <action name="actionImperial">
   <property name="text">
    <string>&amp;Imperial</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Time</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionDistance2D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal Distance</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionTotalSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Total &amp;Speed</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionDiveAngle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dive Angle</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionDistance3D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Total Distance</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionImportGates">
   <property name="text">
    <string>Import &amp;Gates...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGlideRatio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Glide Ratio</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionCurvature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dive &amp;Rate</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Pan</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Measure</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionShowLeftView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Side View</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionShowTopView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>T&amp;op View</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionShowFrontView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Front View</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionZero">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionGround">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Grou&amp;nd</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionHorizontalAccuracy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal Accuracy</string>
   </property>
   <property name="shortcut">
    <string>Shift+H</string>
   </property>
  </action>
  <action name="actionVerticalAccuracy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical Accuracy</string>
   </property>
   <property name="shortcut">
    <string>Shift+V</string>
   </property>
  </action>
  <action name="actionSpeedAccuracy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed Accuracy</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionNumberOfSatellites">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Number Of Satellites</string>
   </property>
   <property name="toolTip">
    <string>Number Of Satellites</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionShowMapView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Map View</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionImportVideo">
   <property name="text">
    <string>Import Vi&amp;deo...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExportKML">
   <property name="text">
    <string>Export &amp;KML...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionExportPlot">
   <property name="text">
    <string>Export P&amp;lot...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionShowWindView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wind View</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionAcceleration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Acceleration</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="actionTotalEnergy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Total Energy</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionEnergyRate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Energy Rate</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionLift">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lift Coefficient</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionDrag">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Drag Coefficient</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionShowLiftDragView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Drag Polar</string>
   </property>
   <property name="shortcut">
    <string>Alt+6</string>
   </property>
  </action>
  <action name="actionShowScoringView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;coring</string>
   </property>
   <property name="shortcut">
    <string>Alt+7</string>
   </property>
  </action>
  <action name="actionOptimize">
   <property name="text">
    <string>Optimize</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionShowOrthoView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>O&amp;rtho View</string>
   </property>
   <property name="toolTip">
    <string>Orthographic View</string>
   </property>
   <property name="shortcut">
    <string>Alt+9</string>
   </property>
  </action>
  <action name="actionExportTrack">
   <property name="text">
    <string>Export Tra&amp;ck...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionShowPlaybackView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Playback</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
  <action name="actionCourse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Course</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionCourseAccuracy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Course Accuracy</string>
   </property>
  </action>
  <action name="actionCourseRate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Course Rate</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionWind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wind Adjustment</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionSetCourse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set C&amp;ourse</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionUndoZoom">
   <property name="text">
    <string>Undo Zoom</string>
   </property>
   <property name="toolTip">
    <string>Undo Zoom</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionRedoZoom">
   <property name="text">
    <string>Redo Zoom</string>
   </property>
   <property name="toolTip">
    <string>Redo Zoom</string>
   </property>
   <property name="shortcut">
    <string>Shift+Backspace</string>
   </property>
  </action>
  <action name="actionShowLogbookView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Logbook</string>
   </property>
   <property name="toolTip">
    <string>Logbook</string>
   </property>
  </action>
  <action name="actionDeleteTrack">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionImportFolder">
   <property name="text">
    <string>Import Folder...</string>
   </property>
  </action>
  <action name="actionZoomToExtent">
   <property name="text">
    <string>Zoom To Extent</string>
   </property>
   <property name="shortcut">
    <string>Shift+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DataPlot</class>
   <extends>QWidget</extends>
   <header>dataplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
